app/api/v2/handlers/fact_api.py

```diff
@@ -67,9 +68,11 @@ async def get_relationships(self, request: web.Request):
                 raise JsonHttpBadRequest(error_msg)
         return web.json_response(dict(found=resp))
 
-    @aiohttp_apispec.docs(tags=['facts'])
+    @aiohttp_apispec.docs(tags=['facts'],
+                          summary='Create facts',
```

Comment:
This endpoint is a little bit confusing because of how the actual endpoint is named, but I think the summary/description can be updated to be `Create fact` as this only creates a single fact